yarn.lock

```diff
@@ -2249,7 +2249,7 @@
   dependencies:
     "@swc/helpers" "^0.5.1"
 
-"@griffel/core@^1.12.2", "@griffel/core@^1.14.1", "@griffel/core@^1.14.3":
+"@griffel/core@^1.14.1", "@griffel/core@^1.14.3":
```

Comment:
Dedupe these?

Reply:
Looks like those dupes a pre-existing and stem from `@fluentui/react-provider` and `@griffel/react`, neither of which come from this PR. Maybe dedupe in a separare PR?

Reply:
<img width="690" alt="image" src="https://github.com/microsoft/fluentui-contrib/assets/14183168/65e078d9-e292-4986-8146-8b3cea92826e">

@spmonahan you won't be able to pass CI without deduping...

Reply:
I guess that's a good reason to fix it now ;)